core/src/main/scala/astraea/spark/rasterframes/RasterFunctions.scala

```diff
@@ -160,6 +160,15 @@ trait RasterFunctions {
   def is_no_data_tile(tile: Column): TypedColumn[Any, Boolean] =
     IsNoDataTile(tile)
 
+  /** Returns true if any cells in the tile are true (non-zero and not nodata). */
+  def exists(tile: Column): TypedColumn[Any, Boolean] = Exists(tile)
+
+  /** Returns true if any cells in the tile are true (non-zero and not nodata). */
+  def any(tile: Column): TypedColumn[Any, Boolean] = Exists(tile)
+
```

Comment:
Why the aliasing? I feel we should just have one function name for the one behavior.

Reply:
I've removed